editor/editor_node.cpp

```diff
@@ -4654,6 +4654,8 @@ void EditorNode::_bind_methods() {
 	ClassDB::bind_method("edit_node", &EditorNode::edit_node);
 	ClassDB::bind_method("_unhandled_input", &EditorNode::_unhandled_input);
 
+	ClassDB::bind_method(D_METHOD("push_item", "object", "property", "inspector_only"), &EditorNode::push_item, DEFVAL(""), DEFVAL(false));
```

Comment:
As stated in [documentation](https://docs.godotengine.org/en/latest/development/cpp/object_class.html#registering-an-object),

> Default values for arguments can be passed in reverse order:
```gdscript
ClassDB::register_method(D_METHOD("methodname", "arg1name", "arg2name"), &MyCustomType::method, DEFVAL(-1)); // default value for arg2name
```

It seems, however, that you passed them in natural order (`DEFVAL("")` for `property`, `DEFVAL(false)` for `inspector_only`).